sgx/report-cache/untrusted/src/lib.rs

```diff
@@ -175,7 +175,24 @@ impl<E: ReportableEnclave, R: RaClient> ReportCache<E, R> {
         let retval = self.ra_client.verify_quote(&quote, Some(ias_nonce))?;
         log::debug!(
             self.logger,
-            "Quote verified by remote attestation service..."
+            "Quote verified by remote attestation service {:?}...",
+            retval,
+        );
+        log::info!(
+            self.logger,
+            "Measurements: MrEnclave: {} MrSigner: {}",
+            VerificationReportData::try_from(&retval)
+                .expect("Could not get verification report data from verification report")
+                .quote
+                .report_body()
+                .expect("Could not get report_body from verification report data")
+                .mr_enclave(),
+            VerificationReportData::try_from(&retval)
+                .expect("Could not get verification report data from verification report")
+                .quote
+                .report_body()
+                .expect("Could not get report_body from verification report data")
+                .mr_signer()
```

Comment:
We shouldn't parse this twice, if at all possible...

Reply:
Sounds good - I wanted to contain it all in the log call - but I'll move the parsing outside. Do you think the expects are ok in this context? Otherwise I'll just log an error.